server/src/main/resources/META-INF/services/org.apache.lucene.codecs.KnnVectorsFormat

```diff
@@ -8,3 +8,4 @@ org.elasticsearch.index.codec.vectors.es816.ES816HnswBinaryQuantizedVectorsForma
 org.elasticsearch.index.codec.vectors.es818.ES818BinaryQuantizedVectorsFormat
 org.elasticsearch.index.codec.vectors.es818.ES818HnswBinaryQuantizedVectorsFormat
```

Comment:
TODO: remove writing from the `ES818HnswFormat` (the ES818 flat format is still used for `bbq_flat`)